### PR TITLE
Fix/campaign front

### PR DESCRIPTION
**Pull request by bitTal**
This PR moves Campaign view to the front-end leaving the Campaign backoffice structure as before.
It sets the backoffice styles to match the main page ones and establishes a common header to share between the main page and the backoffice, changing depending on where the user is.

### CODE REVIEW

File: app/views/admin/events/index.html.erb [hunk 1]
```diff
@@ -16,9 +17,14 @@
     </tr>
   </thead>
 
-  <tbody>
+  <tbody data-role="activerecord_sortable">
```
Comment (tiagojsag) [inline app/views/admin/events/index.html.erb]: @simaob AFAIK only campaigns are sorteable. Can you clarify this please?

**Comment by simaob** [comment]
Yes you are right.

File: app/views/admin/events/index.html.erb (hunk 1 above)
Comment (tiagojsag) [inline app/views/admin/events/index.html.erb]: @bitTal can you revert the changes made to the views other than the campaigns, please?

Comment (bitTal) [inline app/views/admin/events/index.html.erb]: Done.

File: app/views/admin/event_requests/index.html.erb [hunk 2]
```diff
@@ -3,6 +3,7 @@
 <table class="table table-bordered">
   <thead>
     <tr>
+      <th></th>
```
Comment (tiagojsag) [inline app/views/admin/event_requests/index.html.erb]: @bitTal forgot these ;)

Comment (bitTal) [inline app/views/admin/event_requests/index.html.erb]: Removed.

**Comment by tiagojsag** [comment]
Functionally it seems ok, I cant say much about the code itself. Two things:
- This could easily be split into two pull requests, probably more. 48 changed files in a commit is hard to review, please make smaller changes in the future
- We should stop using "git merge master" or "git merge develop" and rebase instead

**Comment by tiagojsag** [comment]
closing in favor of https://github.com/loggingroads/onboarding/pull/11

**Comment by bitTal** [comment]
This PR was created from the feature/back-campaign branch and PR which has been reviewed but never merged. That's why this PR seems to have to many files changed.

**Comment by tiagojsag** [comment]
@bitTal that PR breaks the backend, as we previsouly discussed. it could never be merged. closing it
